virtcontainers/qemu.go

```diff
@@ -838,15 +838,29 @@ func (q *qemu) stopSandbox() error {
 	defer q.cleanupVM()
 	q.Logger().Info("Stopping Sandbox")
 
-	err := q.qmpSetup()
-	if err != nil {
-		return err
-	}
+	pid := q.pid()
 
-	err = q.qmpMonitorCh.qmp.ExecuteQuit(q.qmpMonitorCh.ctx)
-	if err != nil {
-		q.Logger().WithError(err).Error("Fail to execute qmp QUIT")
-		return err
+	signame := "SIGTERM"
+	signum := syscall.SIGTERM
+	for i :=0; i < 75; i++ {
```

Comment:
In general I like the idea, but I have some concerns:

1. Kill QEMU is drastic and prone to data corruption
2. The `SIGKILL` signal is sent after 10 seconds, some container managers will kill the runtime before that happens

Can we keep `ExecuteQuit`, check if QEMU is still running, send the `SIGTERM` signal for a couple of seconds if QEMU is running, and finally send the `SIGKILL` signal if QEMU is still there ?

Reply:
> Kill QEMU is drastic and prone to data corruption
In fact, the sigterm also is graceful shutdown. i would like use SIGTERM instead of qmp quit, as qmp quit depends on qemu's monitor device,  If you want to keep ExecuteQuit, I am also fine to me.
> The SIGKILL signal is sent after 10 seconds, some container managers will kill the runtime before that happens
would be fixed